LICENSE.md

```diff
@@ -1 +1,297 @@
+- **Toronto Social Development, Finance & Administration Department**: source [data](http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=04b489fe9c18b210VgnVCM1000003dd60f89RCRD&vgnextchannel=75d6e03bb8d1e310VgnVCM10000071d60f89RCRD) and [license](http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=4a37e03bb8d1e310VgnVCM10000071d60f89RCRD)
+- **Transitland**: source [data](https://transit.land/) and [license](https://transit.land/an-open-project/contributor-agreement.html)
+- **UN/LOCODE (United Nations Code for Trade and Transport Locations)**: source [data](http://www.unece.org/cefact/locode/welcome.html) and [license](http://www.unece.org/cefact/locode/locode_since1981.html)
+- **U.S. Department of State - Humanitarian Information Unit**: source [data](https://data.humdata.org/dataset/global-lsib-polygons-detailed) and [license](https://data.humdata.org/about/license)
+- **United States Census Bureau**: source [data](https://www.census.gov/) and [license](https://www.census.gov/data/developers/about/terms-of-service.html)
+- **US ZIP Code Tabulation Area**: source [data](http://www.census.gov/geo/reference/zctas.html) and [license](https://www.census.gov/data/developers/about/terms-of-service.html)
+- **Vancouver Planning and Development Services**: source [data](http://data.vancouver.ca/datacatalogue/localAreaBoundary.htm) and [license](http://vancouver.ca/your-government/open-data-catalogue.aspx#tab19099)
+- **Washington Post**: source [data](https://web.archive.org/web/20160209065412/http://opendatadc.org/dataset/neighborhood-boundaries-217-neighborhoods-washpost-justgrimes) and [license](http://opendatadc.org/dataset/neighborhood-boundaries-217-neighborhoods-washpost-justgrimes)
+- **Yahoo! GeoPlanet (formerly Where On Earth)**: source [data](http://developer.yahoo.com/geo/geoplanet/) and [license](http://developer.yahoo.com/geo/geoplanet/data/)
+- **Who's On First**: source [data](https://whosonfirst.org/) and [license](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md)
+- **Wikipedia**: source [data](http://www.wikipedia.org/) and [license](https://en.wikipedia.org/wiki/Wikipedia:Copyrights)
+- **Zetashapes**: source [data](http://www.zetashapes.com) and [license](http://www.zetashapes.com/license). This source includes `CC-BY` compatible data from the following organizations:
+  - **Global**: flickr
+  - **United States**: United States Census Bureau
+- **Zolk Chicago Neighborhoods Map**: source [data](http://chicagomap.zolk.com/) and [license](http://chicagomap.zolk.com/about.html)
+_All 125 sources listed above are currently used to populate Who's On First records or will be added to Who's On First records in the near future._
```

Comment:
The Markdown view appends this onto the same line as Zolk... add more line space?

Reply:
This count should be: **312**

Which is (125 primary sources + 187 additional 'via' sources) and excludes (32 concordance-only sources).

Reply:
both taken care of in 977d240 and 7eef93c